content/inheritance/reading/csharp-inheritance/_index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Inheritance in C#"
+date: 2023-01-31T15:39:38-06:00
+draft: false
+weight: 2
+originalAuthor: Courtney Frey # to be set by page creator
+originalAuthorGitHub: speudusa # to be set by page creator
+reviewer: # to be set by the page reviewer
+reviewerGitHub: # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+Let’s examine an inheritance relationship between two classes: `Cat` and `HouseCat`. `HouseCat` is a class that inherits from Cat, so `HouseCat` receives the data and behaviors of Cat. These inherited traits are things like fields, properties, and methods. Any fields and non-constructor methods in Cat are available to each instance of `HouseCat`.
+
+When we speak about an inheritance relationship, we say that a `HouseCat` is a `Cat`, or **extends** `Cat`. In order to define a class that inherits from another, we use the `:` syntax.
+
+```csharp{linenos=table,hl_lines=[],linenostart=1}
+public class Cat
+{
+   // ...code for the Cat class...
+}
+
+public class HouseCat : Cat
+{
+   // ...code for the HouseCat class...
+}
+```
+
+We say that `HouseCat` is a **subclass**, **derived class**, or **child class** of `Cat`, and we say that `Cat` is the **superclass**, **base class**, or **parent class** of `HouseCat`.
+
+In C#, a class may extend only one class. Classes may extend each other in turn, however. This creates hierarchies of classes. We often visualize these by drawing each class as a box, with an arrow pointing from the subclass to the base class. The image below show that `B` extends `A`.
+
+   {{< rawhtml >}}
+   <img src="../pictures/inheritance-basic.png" alt="B extends A" width=60% />
+   {{< /rawhtml >}}
+
+   The shaded portion of these boxes can include additional information about each class. We’ll learn about what we might put here in a little bit.
+
+Inheritance is an essential mechanism for sharing data and behavior between related classes. Using it effectively creates organized code with groups of classes that have increasingly specialized behavior.
+
+When this happens, we can visualize the inheritance structure with a slightly more complex diagram.
+
+   {{< rawhtml >}}
+   <img src="../pictures/inheritance-tree.png" alt="Inheritance tree with many nodes" width=80% />
+   {{< /rawhtml >}}
```

Comment:
Picture is not showing -> 
<img width="1219" alt="Missing Photo" src="https://user-images.githubusercontent.com/39868164/217950229-1522eb8a-c250-41d1-b584-ecb0509718c2.png">

Picture folder is at the wrong level, problem may be solved by moving the picture folder into the reading folder.

Reply:
fixed. thanks for catching this!